components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -18,10 +18,14 @@ <h2>Add Member Expression</h2>
           <form [formGroup]="expressionForm">
             <chef-form-field>
               <label>Member expression *</label>
-              <input chefInput formControlName="expression" type="text">
+              <input chefInput formControlName="expression" type="text" (keyup)="resetErrors()" >
               <chef-error *ngIf="unparsableMember">Invalid member expression.</chef-error>
               <chef-error *ngIf="duplicateMember">Member expression already in table.</chef-error>
               <chef-error *ngIf="alreadyPolicyMember">Member already in policy.</chef-error>
+                <chef-error *ngIf="(expressionForm.get('expression').hasError('required') || expressionForm.get('expression').hasError('pattern')) && expressionForm.get('expression').dirty">
```

Comment:
We don't incude `dirty` in the other `*ngIf`s -- is that a special thing here?

Reply:
Yes, it is--meant to add a pre-review comment there. The first three can only ever occur by someone actively pressing "Add", so they cannot show up by themselves on page load, and thus no need of `ngIf`. The latter is, like in all the other files, needing of the `dirty` flag so that the error does not show up upon page load.